doc/otel.md

```diff
@@ -0,0 +1,30 @@
+
+# How to setup OTel
+
+Currently only the webhook is instrumented using OTel. To connect the webhook to a tenant do the following steps:

```

Comment:
all entry-points have some level of instrumentation, right ?
yes the webhook has more actual metrics, but every entry-point has some "default" golang ones, right?

if so then this sentence is not that correct 😬

Reply:
You're right, I've added the other entry points after I wrote this doc. Will update.